### ISSUE TITLE
Uncaught TypeError: editor.languageMode.rowRangeForParagraphAtBufferRow is not a function

### ISSUE THREAD

**Issue by sethvargo** [forivall/atom-hard-wrap#51]
1. Choose menu option

**Atom**: 1.22.0 x64
**Electron**: 1.6.15
**OS**: Mac OS X 10.13.1
**Thrown From**: [hard-wrap](https://github.com/forivall/atom-hard-wrap) package 1.1.0

### Stack Trace

Uncaught TypeError: editor.languageMode.rowRangeForParagraphAtBufferRow is not a function

```
At /Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/text-buffer.js:959

TypeError: editor.languageMode.rowRangeForParagraphAtBufferRow is not a function
    at /packages/hard-wrap/lib/hard-wrap.js:81:13
    at TextBuffer.module.exports.TextBuffer.transact (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/text-buffer.js:952:24)
    at TextEditor.module.exports.TextEditor.transact (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:1687:32)
    at Object.reflowSelection (/packages/hard-wrap/lib/hard-wrap.js:75:12)
    at HTMLElement.hardWrapReflowSelection (/packages/hard-wrap/lib/hard-wrap.js:23:48)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:381:36)
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:621:22)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:412:28)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:100:42)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:3:65)
```

### Commands

```
 20x -1:48.6.0 core:select-left (input.hidden-input)
     -1:47.5.0 core:cut (input.hidden-input)
     -1:47.4.0 core:backspace (input.hidden-input)
 17x -1:47.1.0 core:move-left (input.hidden-input)
     -1:46.2.0 core:move-right (input.hidden-input)
     -1:46 core:paste (input.hidden-input)
     -1:45.5.0 core:backspace (input.hidden-input)
     -1:45.1.0 core:save (input.hidden-input)
     -0:34.2.0 pane:show-previous-item (input.hidden-input)
  2x -0:21.9.0 core:move-right (input.hidden-input)
     -0:21.1.0 core:move-left (input.hidden-input)
     -0:17.2.0 hard-wrap:reflow-selection (input.hidden-input)
     -0:14.7.0 editor:consolidate-selections (input.hidden-input)
     -0:14.7.0 core:cancel (input.hidden-input)
     -0:14.3.0 core:save (input.hidden-input)
     -0:13.2.0 hard-wrap:reflow-selection (input.hidden-input)
```

### Non-Core Packages

```
busy-signal 1.4.3 
file-icons 2.1.13 
go-debug 1.5.2 
go-plus 5.6.0 
go-signature-statusbar 1.2.3 
hard-wrap 1.1.0 
hyperclick 0.1.5 
intentions 1.1.5 
language-docker 1.1.8 
language-nginx 0.8.0 
linter 2.2.0 
linter-ui-default 1.6.10 
sort-lines 0.18.0 
```

**Comment by jikk** [comment]
I am also experiencing the same issue.

**Comment by mehaase** [comment]
This is a duplicate of #50 (which unfortunately does not mention the error message).

**Comment by mhow2** [comment]
Hi, experiencing this too. Is it going to be fixed ? Regards.

**Comment by mehaase** [comment]
@mhow2 This repo hasn't been updated in over a year. I guess the project is dead. But there is a patch in #50 that fixes this particular issue.

**Comment by forivall** [comment]
t'would have been nicer if #50 was a PR instead of just a patch pasted in a comment, then i would have noticed

yeah, i'm mostly using vs code these days, but i'm willing to take pull requests and transfer ownership to someone who is willing to maintain

**Comment by forivall** [comment]
fixed and released in 1.7.0
